pkg/builder/builder.go

```diff
@@ -735,6 +781,37 @@ func makeNMAContainer(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.Container
 	}
 }
 
+// makeScrutinizeInitContainer builds the spec of the init container that collects
+// scrutinize command
+func makeScrutinizeInitContainer(vscr *v1beta1.VerticaScrutinize) corev1.Container {
+	return corev1.Container{
+		Image: "busybox:latest",
+		Name:  names.ScrutinizeInitContainer,
+		Command: []string{
+			"sh",
+			"-c",
+			"sleep 5",
+		},
+		VolumeMounts: []corev1.VolumeMount{buildScrutinizeVolumeMount(vscr)},
+		Resources:    vscr.Spec.Resources,
+	}
+}
+
+// makeScrutinizeMainContainer builds the spec of the container that will
+// be running after all init containers are completed
+func makeScrutinizeMainContainer(vscr *v1beta1.VerticaScrutinize) corev1.Container {
+	return corev1.Container{
+		Image: "busybox:latest",
+		Name:  names.ScrutinizeMainContainer,
+		Command: []string{
```

Comment:
It would be good that when you exec into the main container you are in the directory where the scrutinize tar ball exists. This probably depends on what image we ultimately use here. Busy box may be okay.

Reply:
Were you able to do anything for this comment?

Reply:
Yes I pushed the wrong file. Now you see it.